Sign_in.feature

```diff
@@ -0,0 +1,41 @@
+Feature: Sign in to the Saucedemo Application
+
+As a user, I want to sign in so I can use application functionality.
+
+Background:
+  Given the application is open at "https://www.saucedemo.com/"  
+
+  # Positive Scenarios
+  Scenario: Standard user signs in successfully
+    When the standard user signs in with username "standard_user" and password "secret_sauce"
```

Comment:
Need to write them as separate steps: entered "..." username; entered "...." password"